test/src/eks_blueprints_e2e_test.go

```diff
@@ -0,0 +1,325 @@
+//go:build e2e
+// +build e2e
+
+package src
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+	"testing"
+	internal "github.com/aws-ia/terraform-aws-eks-blueprints/aws"
+)
+
+var (
+	//Test Driven tests Inputs https://github.com/golang/go/wiki/TableDrivenTests
+	testCases = []struct {
```

Comment:
should we type this struct?

Reply:
Just using the standard table driven struct from Go tests. I will explore this bit more in my next improvement PR